Lib/math/__init__.py

```diff
@@ -0,0 +1,19 @@
+"""
+math module -- Mathematical functions
+"""
```

Comment:
Is changing the docstring intentional?
```suggestion
"""
This module provides access to the mathematical functions
defined by the C standard.
"""
```